.github/workflows/Tutor_CI.yml

```diff
@@ -13,8 +13,15 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: 3.7
+    - name: Install dependencies(flake8 and pytest included)
+      run: |
+        chmod +x ./bootstrap.sh
+        ./bootstrap.sh
     - name: Lint with flake8
       run: |
         pip install flake8
-        flake8 . --count --select=W291,W292,W293,E3,E9 --show-source --statistics
+        flake8 . --count --select=W292,W293,E3,E9 --show-source --statistics
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```

Comment:
You might want to remove the '--exit-zero' flag

Reply:
@lioramilbaum removing that flag, break the CI and we got errors on the lint section.

Reply:
Exactly. Those issues should be addressed. Don't you think so?

Reply:
On the first line we chose the specific issues we want to handle. A lot of the issues are breaking the CI is like `imported but unused` which is not true. That's why we changed it on #55 .
If we should handle all the issues - why we need to first line?

Reply:
Good question. The default should be handling all issues. There should be a good reason for handling just a few.

Reply:
@lioramilbaum On #81 I fixed all the relevant issues, that are not on the first line(most of them about indentation and unused importing). There are issues related to importing(which unused but without them the app isn't working(like routes importing) and `migrations` folder issues that not related to us - `flake-migrate` responsible for generating those files. I tried to exclude it but no matter what I try - it still countable. 
Those specific options chose because there are errors that are not under our control.

Reply:
There are ways to tell flake8 to skip a line. For example: Using the '# noqa' stanza.

Reply:
Done. Now I'm looking for a way to make it skip migrations folder. Using `--exclude` flag or `setup.cfg` file do nothing.

Reply:
On #82 I fixed all the linting problems, and as you can see - there are no linting problems. Also, updated that PR to exclude `venv` and `migrations` folder and remove `exit-zero` flag.